bin/specs-to-refactor

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env ruby
+#
+# This script analyses the Placements system specs and lists
+# those which have already been refactored to Cucumber-style specs,
+# and those which still need to be refactored.
+#
+# It produces Markdown format output so it's easy to copy-and-paste into the
+# Trello card: https://trello.com/c/TayQl2rn/244-refactor-system-specs-to-conform-to-new-style-guide
+
+@root_path = File.realpath(File.dirname(__FILE__) + "/..")
+system_specs = Dir["#{@root_path}/spec/system/placements/**/*_spec.rb"]
+
+# Find specs which contain "scenario do" – i.e. a scenario with no description.
```

Comment:
javascript-enabled specs have a scenario name in order to allow the `:js` tag to be added, so they won't show up in the cucumber count.

Reply:
We should be able to put the `:js` tag on the overarching `RSpec.describe` block, like this:

https://github.com/DFE-Digital/itt-mentor-services/blob/864c40d76ea25f45aecd09bfa37fef2ab8ae8d10/spec/system/placements/schools/school_contacts/user_adds_a_placement_contact_with_multiple_tabs_spec.rb#L3-L4